aggregatebatteries.py

```diff
@@ -563,11 +563,9 @@ def _update(self):
        
                 if self._balancing == 1:
                     ChargeVoltageBattery = CVL_BALANCING
-                    if (Voltage >= 0.999 * CVL_BALANCING):
-                        self._ownCharge = InstalledCapacity                                         # reset Coulumb counter to 100%
-                        if ((MaxCellVoltage - MinCellVoltage) < CELL_DIFF_MAX):
-                            self._balancing = 2;
-                            logging.info('%s: Balancing goal reached.'  % (dt.now()).strftime('%c'))    
+                    if (Voltage >= CVL_BALANCING) and ((MaxCellVoltage - MinCellVoltage) < CELL_DIFF_MAX):
+                        self._balancing = 2;
+                        logging.info('%s: Balancing goal reached.'  % (dt.now()).strftime('%c'))    
```

Comment:
Here are some changes I did not made. Did you push a commit maybe between my fork and the merge of the PR?